src/legacy/ui/public/directives/_saved_object_paginated_selectable.scss

```diff
@@ -1,70 +0,0 @@
-saved-object-finder,
```

Comment:
This selector is still being used by Timelion

Reply:
How exactly? This file is theoretically not imported from anywhere.

Reply:
@spalger Are the scss files from `ui/public` still shipped into the theme files?

Reply:
@elastic/kibana-app After checking with Tyler and Spencer, it seems like this file is already not used on current master, as the legacy scss files are no longer imported. Could you confirm that this file is safe to delete (or should I move it to the timelion plugin)?

Reply:
It looks like the removal of the import was not intentional as it did remove the styles from where the this component lives. I would move the styles into the Timelion application. But you can remove the second selector of `paginated-selectable-list`

Reply:
Done in https://github.com/elastic/kibana/pull/76085/commits/925942f487211adb2e75b9afbcd4beeaf5d899c8. Note that because of the sasslint rules, I had to remove the `!important` from the styles, but it seems they were useless, as even without `!important`, they are the highest priority selectors inside the component.